src/main/java/de/cas/mse/exercise/diamond/Diamond.java

```diff
@@ -4,28 +4,28 @@
 
 public class Diamond {
 
-	public String print(int n) {
-		if (n <= 0 || n % 2 == 0) {
+	public String createDiamond(int maximumDimension) {
```

Comment:
createDiamond: It's debatable whether this API change worth it. this name feels for us like a factory method.
maximumDimension: Is it only a restriction? Can the method create a smaller diamond as well?